Umami/README.md

```diff
@@ -0,0 +1,42 @@
+# Install Umami on Codesphere
+Hey folks,
+Have you ever wondered whether you can host your own Google Analytics alternative (because Google collects a lot of data from it)?
+
+If yes, you’re on the very correct blog post!
+
+Let’s dive into how we can deploy our own Analytics service using Codesphere Cloud!
+
+## Steps for installing Umami on Codesphere.
+1. Signup On Codesphere — https://codesphere.com/
+2. Create a new instance using the “+ New Workspace” button.
+3. Put in the URL: https://github.com/sancho1952007/install-umami-on-codesphere.git in the input box.
```

Comment:
Instead of a reference to a repository of yours, change it to https://github.com/codesphere-community/umami

Reply:
Done ✅